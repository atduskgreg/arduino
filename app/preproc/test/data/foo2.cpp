void playMessage(prog_uchar* message){
  
maxStringSize =  sizeof(message);  // find message size
}

void loop() {
  playMessage(signMessage);
}
