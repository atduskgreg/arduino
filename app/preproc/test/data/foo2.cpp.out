void playMessage(prog_uchar* message);
void loop();
